### DISCUSSION TITLE
About Minecraft 16.5 Compatibility

### DISCUSSION THREAD

**Discussion by jotaha** [PulseBeat02/mcav#29] [Q&A] [answered]
Is there a specific reason 16.5 support was removed?

Are there any improvements or features that would not be compatible with that version?

**Comment by PulseBeat02** [comment, marked as answer]
1.16.5 support was removed because it uses a ton of Reflection calls in the code. In 1.17, a lot of packet definition has changed allowing you to define a packet directly instead of using Reflection to define, it meaning it is much faster. Since the plan is to really only support newer versions (and also it's a ton more work to support anything < 1.17 because of the changes), the goal is to only support 1.17+.
